fe/fe-core/src/test/java/com/starrocks/analysis/CreateTableStmtTest.java

```diff
@@ -235,6 +235,29 @@ public void testArrayKey() throws Exception {
 
     @Test
     public void testBitmapWithoutAggregateMethod() throws Exception {
+        ColumnDef col3 = new ColumnDef("col3", new TypeDef(ScalarType.createType(PrimitiveType.BITMAP)));
+        cols.add(col3);
+        CreateTableStmt stmt = new CreateTableStmt(false, false, tblNameNoDb, cols, "olap",
+                new KeysDesc(KeysType.AGG_KEYS, colsName), null,
+                hashDistributioin, null, null, "");
+        expectedEx.expect(AnalysisException.class);
+        expectedEx.expectMessage("AGG_KEYS table should specify aggregate type for non-key column[col3]");
+        stmt.analyze(analyzer);
+    }
+
+    @Test
+    public void testBitmapWithPrimaryKey() throws Exception {
+        ColumnDef col3 = new ColumnDef("col3", new TypeDef(ScalarType.createType(PrimitiveType.BITMAP)));
+        cols.add(col3);
+        CreateTableStmt stmt = new CreateTableStmt(false, false, tblNameNoDb, cols, "olap",
+                new KeysDesc(KeysType.PRIMARY_KEYS, colsName), null,
+                hashDistributioin, null, null, "");
+        expectedEx.expect(AnalysisException.class);
```

Comment:
why UniqueKey and PrimaryKey are different

Reply:
SLF4J: Actual binding is of type [org.slf4j.impl.Log4jLoggerFactory]
`col1` int(11) NOT NULL COMMENT ""
`col2` char(10) NOT NULL COMMENT ""
`col3` bitmap NOT NULL COMMENT ""
[ERROR] Tests run: 21, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 6.017 s <<< FAILURE! - in com.starrocks.analysis.CreateTableStmtTest
[ERROR] testBitmapWithPrimaryKey(com.starrocks.analysis.CreateTableStmtTest)  Time elapsed: 0.24 s  <<< ERROR!
com.starrocks.common.AnalysisException: primary key column[col2] type not supported: char(10)
	at com.starrocks.analysis.CreateTableStmtTest.testBitmapWithPrimaryKey(CreateTableStmtTest.java:258)

because of this error, I will fix it